bin/cml/runner.js

```diff
@@ -533,9 +533,9 @@ exports.builder = (yargs) =>
       },
       cloudGpu: {
         type: 'string',
-        choices: ['nogpu', 'k80', 'v100', 'tesla'],
-        coerce: (val) => (val === 'nogpu' ? undefined : val),
-        description: 'GPU type.'
+        description:
+          'GPU type. Choices: [k80, v100]. Also supports native types like e.g. nvidia-tesla-t4',
```

Comment:
```suggestion
          'GPU type. Choices: nogpu (default), t4, k80, v100, or native types e.g. nvidia-tesla-t4',
```

Reply:
Do we really have to include `nogpu` in that list?

Reply:
Also, `t4` is not supported on `iterative_machine`

Reply:
> Do we really have to include `nogpu` in that list?

I've seen people do `--reuse=false` so... maybe.

> Also, `t4` is not supported on `iterative_machine`

what is `iterative_machine` used for? `iterative_task` and `iterative_runner` are the only things I [unfortunately know of](https://github.com/iterative/cml/issues/1039#issuecomment-1144701993).

Reply:
> _I've seen people do --reuse=false so... maybe._

I've seen people do `--cloud $(base64 --decode <<< YXdz)` and it works, but that doesn't mean we should document it. 😉

Reply:
k feel free to merge unresolved.